stubs/pywin32/win32/lib/win32pdhquery.pyi

```diff
@@ -24,6 +25,8 @@ class BaseQuery:
 class Query(BaseQuery):
     volatilecounters: Incomplete
     def __init__(self, *args, **namedargs) -> None: ...
+    @deprecated("Use of addcounterbybrowsing instead as its considerably more flexible and general.")
```

Comment:
```suggestion
    @deprecated("Use of addcounterbybrowsing instead as it is considerably more flexible and general.")
```
Or just omit the part after "as"; I feel it's not actually helpful to users

Reply:
```suggestion
    @deprecated("Use `addcounterbybrowsing` instead.")
```